source/MRViewer/MRObjectHighlighter.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "MRMesh/MRMeshFwd.h"
+#include "MRViewer/exports.h"
+
+namespace MR
+{
+
+// This is used to temporarily change object colors (and other visual properties) in the scene.
+// NOTE: Currently at most one highlighter can operate at a time.
+// The workflow is as follows:
+// 1. Create `ObjectHighlighter` as a class member in your plugin.
+// 2. Using the constructor parameters, you can customize which fields we roll back. The default behavior should usually be good enough.
+// 3. Call `updateColors()` every frame in `preDraw_()` to set the desired colors.
```

Comment:
user should do it him-self? better to subscribe this class and reduce user calls

Reply:
The idea is to be able to share one highlighter between multiple widgets.

Reply:
The idea is to be able to share a highlighter between multiple widgets. Not sure how to do it otherwise.